### ISSUE TITLE
Wifi issue

### ISSUE THREAD

**Issue by ws839750375** [Daniel203/Thinkbook13s-Hackintosh#1]
Hey,i think Airportltlwm.kext xould let the wifi work. Could you have a try？

**Comment by Daniel203** [comment]
Hi, first of all thank you for writing to me.

I had tried before and the wifi card was not recognized. 
I tried again today after seeing your message and the card is recognized and can connect to the wifi.

The problem though is that the computer can't access the internet despite being connected to the wifi. Basically when I open an internet page it tells me that it can't connect to the internet.

In these days I'll try to see if somehow I can fix.

**Comment by ws839750375** [comment]
Hi,

i'm using the EFI from another reponstory, i think you could have try , everything is good, and it also compatible with your device 13S https://github.com/cupugila/Thinkbook-13s-IWL-EFI-Hackintosh . i think you could try it  first :)

**Comment by Daniel203** [comment]
So, after a few tries, both with my configuration and the one you recommended, I was able to connect to the wifi, however, it only works if the wifi is connected to 2.4ghz.
When the pc is connected to the 5ghz the pages and applications do not load, even if the operating system shows that it is correctly connected to the network.

**Comment by ws839750375** [comment]
For me, 5G works very well. Even our office has wifi roaming enabled on different floors. It works very well. Is the WIFI frequency beyond the range supported by the network card?

**Comment by Daniel203** [comment]
Yes, I was definitely in the range supported by the network card, because the PC was 2-3 meters from the modem, without any wall in between.
When I moved away and the PC connected to the 2.4ghz network, the connection worked properly.
To confirm this, disabling 5ghz directly from the modem settings, everything worked properly.
I've also tried changing kext versions, but the problem seems to persist.

However, maybe I found a better solution. Instead of installing the kext **airportitlwm**, I installed **itlwm** and used heliport, and this seems to work correctly, in both 2.4ghz and 5ghz.

I haven't used it much yet, so I haven't had a chance to test on multiple different connections, but in the next few days, I'll try to see how stable it is.
